### ISSUE TITLE
A11Y: Images/videos in media gallery not announced as clickable

### ISSUE THREAD

**Issue by dreamofabear** [GoogleForCreators/web-stories-wp#4429]
## Bug Description

Images and videos in the media gallery should a clickable ARIA role e.g. `button` or similar.

## Screenshots

<img width="251" alt="Screen Shot 2020-09-09 at 4 20 24 PM" src="https://user-images.githubusercontent.com/20525523/92649074-7a6ba280-f2b8-11ea-95a8-f6cd7239e3f3.png">

## Additional Context

- Plugin Version: 1.0.0-beta.2+fd68af7
- Operating System: macOS 10.15.6
- Browser: Chrome 85

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
